<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Таблица рецептов</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="11" colspan="5">
     <widget class="QLineEdit" name="lineEditURL">
      <property name="font">
       <font>
        <family>Bahnschrift Light</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="8" column="11" colspan="5">
     <widget class="QPushButton" name="pushButtonSingleGet">
      <property name="font">
       <font>
        <family>Bahnschrift Light</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Получить рецепт по ссылке</string>
      </property>
     </widget>
    </item>
    <item row="24" column="0" colspan="16">
     <widget class="QTableView" name="tableView">
      <property name="font">
       <font>
        <family>Bahnschrift Light</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item row="1" column="11">
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift Light</family>
        <pointsize>14</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Вставьте ссылку:</string>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse</set>
      </property>
     </widget>
    </item>
    <item row="28" column="6">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ссылок на рецепт: </string>
      </property>
     </widget>
    </item>
    <item row="29" column="6">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Записей в таблице:</string>
      </property>
     </widget>
    </item>
    <item row="28" column="0" rowspan="2" colspan="4">
     <widget class="QPushButton" name="pushButtonDelete">
      <property name="font">
       <font>
        <family>Bahnschrift Light</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Удалить выбранный рецепт</string>
      </property>
     </widget>
    </item>
    <item row="28" column="11" rowspan="2" colspan="2">
     <widget class="QPushButton" name="pushButtonScanURLs">
      <property name="font">
       <font>
        <family>Bahnschrift Light</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Обновить/создать список ссылок</string>
      </property>
     </widget>
    </item>
    <item row="28" column="13" rowspan="2">
     <widget class="QPushButton" name="pushButtonMultiGet">
      <property name="font">
       <font>
        <family>Bahnschrift Light</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Занести все ссылки в базу</string>
      </property>
     </widget>
    </item>
    <item row="28" column="14" rowspan="2" colspan="2">
     <widget class="QPushButton" name="pushButtonStop">
      <property name="font">
       <font>
        <family>Bahnschrift Light</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Остановить</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>Bahnschrift Light</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Поиск по таблице:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="6">
     <widget class="QLineEdit" name="lineEditSearch">
      <property name="font">
       <font>
        <family>Bahnschrift Light</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="6">
     <widget class="QPushButton" name="pushButtonFind">
      <property name="font">
       <font>
        <family>Bahnschrift Light</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Найти</string>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QRadioButton" name="radioButtonName">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift Light</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Название блюда</string>
      </property>
     </widget>
    </item>
    <item row="8" column="6">
     <widget class="QRadioButton" name="radioButtonIngredients">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift Light</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ингредиенты</string>
      </property>
     </widget>
    </item>
    <item row="28" column="7" colspan="4">
     <widget class="QLabel" name="labelCount">
      <property name="font">
       <font>
        <family>Bahnschrift Light</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="29" column="7" colspan="4">
     <widget class="QLabel" name="labelCountRecipes">
      <property name="font">
       <font>
        <family>Bahnschrift Light</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="28" column="4" rowspan="2">
     <widget class="QPushButton" name="pushButtonMultiDelete">
      <property name="font">
       <font>
        <family>Bahnschrift Light</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Удалить по фильтру</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
